src/main.js

```diff
@@ -972,7 +973,8 @@ function gotoServerValidateUrl(id, guid, url) {
                     if (gotoServerMiddleClick) {
                         window.open(verifyURL(finalURL), '_blank');
                         gotoServerMiddleClick = false;
-                        document.getElementById('gotoServerResponse').innerHTML = '<b>Opening new window to ' + NETDATA.registry.machines[guid].name + '<br/><a href="' + verifyURL(finalURL) + '">' + escapeUserInputHTML(url) + '</a></b><br/>(check your pop-up blocker if it fails)';
+                        const registryMachines = getFromRegistry("registryMachines");
+                        document.getElementById('gotoServerResponse').innerHTML = '<b>Opening new window to ' + registryMachines[guid].name + '<br/><a href="' + verifyURL(finalURL) + '">' + escapeUserInputHTML(url) + '</a></b><br/>(check your pop-up blocker if it fails)';
```

Comment:
Could this markdown be wrapped into template or function?

Reply:
yeah, i agree completely, but we have lots of strings like this in main.js... we'll remove all of them someday when we'll clean (remove) the whole main.js file

Reply:
Oh, ok!